dashboard/config/libraries/Outbreak.interpreted.js

```diff
@@ -0,0 +1,158 @@
+/* -------------- START OF MONSTER BEHAVIOR DEFINITIONS -------------- */
```

Comment:
also for my own understanding -- how did you create this library?

Reply:
`https://levelbuilder-studio.code.org/libraries/new`